pmm-app/src/pmm-dbaas/components/DBCluster/AddDBClusterModal/DBClusterBasicOptions/DBClusterBasicOptions.tsx

```diff
@@ -23,12 +20,20 @@ export const DBClusterBasicOptions: FC<DBClusterBasicOptionsProps> = ({
     change(AddDBClusterFields.databaseType, databaseType);
   }, []);
 
+  const kubernetesClusterName = useCallback((value: string) => {
```

Comment:
nit: add function to validators (either shared or platform-core) to accept regex as parameter and message to return in case of error so we can reuse it.

Reply:
thanks, created task for it PMM-7036